data_sets/yymnist_test.json

```diff
@@ -0,0 +1,30 @@
+{
+    "data_type": "image",
+    "description": "Juneberry metadata files created using detectron2 text detection.",
```

Comment:
Not text detection. :)

Reply:
Adjusted descriptions in this commit 3c344c0